templates/resources/resource_form.html

```diff
@@ -7,13 +7,13 @@
 {% block main_content %}
   <div class="form-holder">
     <p>We hope all changemakers consider publicly sharing the resources they find most useful! Together, we can grow our 
-      collective knowledge and skill up our community. You can submit your resource here. When you submit a resource, it goes 
-      to the Librarian for approval. If there is a problem with your resource, the Librarian will reach out to you directly. 
-      If everything is in order, your resource will be published. When your resource is published on the Commons Library, it 
-      still belongs to you. You can edit its content and privacy settings at any time through the site, and remove it completely
-      by emailing the librarian at librarian@commonslibrary.org The Commons will honour your privacy selection, whether your resource 
-      is publicly available or can be seen only by members of your group. The Commons is not liable for any copying or further distribution 
-      that may occur once your resource has been shared on the site.</p>
+      collective knowledge and skill up our community. You can submit your resource here.</br></br>
+      When you submit a resource, it goes to the Librarian for approval. If there is a problem with your resource, the Librarian will reach out to you directly. 
+      If everything is in order, your resource will be published.</br></br>
+      When your resource is published on the Commons Library, it still belongs to you. You can edit its content and privacy settings at any time through the site, and remove it completely
+      by emailing the librarian at <a href="mailto:librarian@commonslibrary.org">librarian@commonslibrary.org</a></br></br>
+      The Commons will honour your privacy selection, whether your resourcee is publicly available or can be seen only by members of your group. The Commons is not liable for any copying 
+      or further distribution that may occur once your resource has been shared on the site.</p>
```

Comment:
Can we add another `</br>` after the last `p` tag 🙏
Tis a slightly hacky but harmless way to add padding between the description and form 🙂